server/transcription_api.go

```diff
@@ -51,9 +51,18 @@ func (p *Plugin) transcriptionJobTimeoutChecker(callID, jobID string) {
 		clientState.EndAt = time.Now().UnixMilli()
 
 		if state.Call.Recording != nil && state.Call.Recording.EndAt == 0 {
+			recClientState := state.Call.Recording.getClientState()
 			if _, _, err := p.stopRecordingJob(state, callID); err != nil {
-				p.LogError("failed to stop recording job", "callID", callID, "err", err.Error())
+				p.LogError("failed to stop recording job", "err", err.Error(), "callID", callID, "jobID", jobID)
 			}
+
+			// This is needed as we don't yet handle wsEventCallTranscriptionState on
+			// the client since jobs are coupled.
+			recClientState.Err = "failed to start transcriber job: timed out waiting for bot to join call"
+			p.publishWebSocketEvent(wsEventCallRecordingState, map[string]interface{}{
+				"callID":   callID,
+				"recState": clientState.toMap(),
+			}, &model.WebsocketBroadcast{ChannelId: callID, ReliableClusterSend: true})
```

Comment:
This is something I forgot to add originally. If the transcribing job fails first then we wouldn't notify the host as we are not handling the below `wsEventCallTranscriptionState` event.